{
  "uid" : "fdc1287c69e94cff",
  "name" : "test_get_login_submit",
  "fullName" : "test_testcase.TestSingle#test_get_login_submit",
  "historyId" : "9d45b30fd97667aff13f3a162735a3a9",
  "time" : {
    "start" : 1576601905094,
    "stop" : 1576601905455,
    "duration" : 361
  },
  "description" : " 呵呵呵呵1234",
  "descriptionHtml" : "<p>呵呵呵呵1234</p>\n",
  "status" : "failed",
  "statusMessage" : "AssertionError: assert '2' == '0'\n  - 2\n  + 0",
  "statusTrace" : "self = <tests.test_testcase.TestSingle object at 0x105fd7ac8>\n\n    @allure.story(\"测试3\")\n    def test_get_login_submit(self):\n        \"\"\" 呵呵呵呵1234\"\"\"\n        single_testcase_yaml = os.path.join(os.path.dirname(__file__), \"api\", \"get_login_submit.yml\")\n    \n        result = self.run.run_yml(single_testcase_yaml)\n        logger.info(\"666666666\"+str(result))\n        for ass in result[0]:\n            # logger.info(ass[\"url\"])\n            # logger.info(\"\\033[1;31m=======Assert=====\\033[0m\")\n            # logger.info(\"\\033[1;36mexpected:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"expected\", ass[\"expected\"]))\n            # logger.info(\"\\033[1;32mactual:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"actual\", ass[\"actual\"]))\n            logger.info(\"hehjehehehhhjhjhjhejhjehjehje\")\n            logger.info(ass[\"url\"])\n            logger.info(\"=======Assert=====\")\n            logger.info(\"expected:\" + '\\n\\t' + \"{}={}\".format(\"expected\", ass[\"expected\"]))\n            logger.info(\"actual:\" + '\\n\\t' + \"{}={}\".format(\"actual\", ass[\"actual\"]))\n            try:\n>               assert ass[\"expected\"] == ass[\"actual\"]\nE               AssertionError: assert '2' == '0'\nE                 - 2\nE                 + 0\n\ntest_testcase.py:79: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.test_testcase.TestSingle object at 0x105fd7ac8>\n\n    @allure.story(\"测试3\")\n    def test_get_login_submit(self):\n        \"\"\" 呵呵呵呵1234\"\"\"\n        single_testcase_yaml = os.path.join(os.path.dirname(__file__), \"api\", \"get_login_submit.yml\")\n    \n        result = self.run.run_yml(single_testcase_yaml)\n        logger.info(\"666666666\"+str(result))\n        for ass in result[0]:\n            # logger.info(ass[\"url\"])\n            # logger.info(\"\\033[1;31m=======Assert=====\\033[0m\")\n            # logger.info(\"\\033[1;36mexpected:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"expected\", ass[\"expected\"]))\n            # logger.info(\"\\033[1;32mactual:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"actual\", ass[\"actual\"]))\n            logger.info(\"hehjehehehhhjhjhjhejhjehjehje\")\n            logger.info(ass[\"url\"])\n            logger.info(\"=======Assert=====\")\n            logger.info(\"expected:\" + '\\n\\t' + \"{}={}\".format(\"expected\", ass[\"expected\"]))\n            logger.info(\"actual:\" + '\\n\\t' + \"{}={}\".format(\"actual\", ass[\"actual\"]))\n            try:\n                assert ass[\"expected\"] == ass[\"actual\"]\n            except AssertionError:\n>               assert ass[\"expected\"] == ass[\"actual\"]\nE               AssertionError: assert '2' == '0'\nE                 - 2\nE                 + 0\n\ntest_testcase.py:81: AssertionError",
  "flaky" : false,
  "beforeStages" : [ {
    "name" : "_Module__pytest_setup_module",
    "time" : {
      "start" : 1576601904023,
      "stop" : 1576601904024,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_Class__pytest_setup_class",
    "time" : {
      "start" : 1576601904024,
      "stop" : 1576601904024,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "description" : " 呵呵呵呵1234",
    "status" : "failed",
    "statusMessage" : "AssertionError: assert '2' == '0'\n  - 2\n  + 0",
    "statusTrace" : "self = <tests.test_testcase.TestSingle object at 0x105fd7ac8>\n\n    @allure.story(\"测试3\")\n    def test_get_login_submit(self):\n        \"\"\" 呵呵呵呵1234\"\"\"\n        single_testcase_yaml = os.path.join(os.path.dirname(__file__), \"api\", \"get_login_submit.yml\")\n    \n        result = self.run.run_yml(single_testcase_yaml)\n        logger.info(\"666666666\"+str(result))\n        for ass in result[0]:\n            # logger.info(ass[\"url\"])\n            # logger.info(\"\\033[1;31m=======Assert=====\\033[0m\")\n            # logger.info(\"\\033[1;36mexpected:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"expected\", ass[\"expected\"]))\n            # logger.info(\"\\033[1;32mactual:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"actual\", ass[\"actual\"]))\n            logger.info(\"hehjehehehhhjhjhjhejhjehjehje\")\n            logger.info(ass[\"url\"])\n            logger.info(\"=======Assert=====\")\n            logger.info(\"expected:\" + '\\n\\t' + \"{}={}\".format(\"expected\", ass[\"expected\"]))\n            logger.info(\"actual:\" + '\\n\\t' + \"{}={}\".format(\"actual\", ass[\"actual\"]))\n            try:\n>               assert ass[\"expected\"] == ass[\"actual\"]\nE               AssertionError: assert '2' == '0'\nE                 - 2\nE                 + 0\n\ntest_testcase.py:79: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.test_testcase.TestSingle object at 0x105fd7ac8>\n\n    @allure.story(\"测试3\")\n    def test_get_login_submit(self):\n        \"\"\" 呵呵呵呵1234\"\"\"\n        single_testcase_yaml = os.path.join(os.path.dirname(__file__), \"api\", \"get_login_submit.yml\")\n    \n        result = self.run.run_yml(single_testcase_yaml)\n        logger.info(\"666666666\"+str(result))\n        for ass in result[0]:\n            # logger.info(ass[\"url\"])\n            # logger.info(\"\\033[1;31m=======Assert=====\\033[0m\")\n            # logger.info(\"\\033[1;36mexpected:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"expected\", ass[\"expected\"]))\n            # logger.info(\"\\033[1;32mactual:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"actual\", ass[\"actual\"]))\n            logger.info(\"hehjehehehhhjhjhjhejhjehjehje\")\n            logger.info(ass[\"url\"])\n            logger.info(\"=======Assert=====\")\n            logger.info(\"expected:\" + '\\n\\t' + \"{}={}\".format(\"expected\", ass[\"expected\"]))\n            logger.info(\"actual:\" + '\\n\\t' + \"{}={}\".format(\"actual\", ass[\"actual\"]))\n            try:\n                assert ass[\"expected\"] == ass[\"actual\"]\n            except AssertionError:\n>               assert ass[\"expected\"] == ass[\"actual\"]\nE               AssertionError: assert '2' == '0'\nE                 - 2\nE                 + 0\n\ntest_testcase.py:81: AssertionError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "6d7d900f69c58f22",
      "name" : "log",
      "source" : "6d7d900f69c58f22.txt",
      "type" : "text/plain",
      "size" : 2256
    }, {
      "uid" : "ea1e7d4fd4615dcf",
      "name" : "stdout",
      "source" : "ea1e7d4fd4615dcf.txt",
      "type" : "text/plain",
      "size" : 0
    }, {
      "uid" : "634d4ae07f287270",
      "name" : "stderr",
      "source" : "634d4ae07f287270.txt",
      "type" : "text/plain",
      "size" : 0
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 3,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "_Module__pytest_setup_module::0",
    "time" : {
      "start" : 1576601905891,
      "stop" : 1576601905891,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_Class__pytest_setup_class::0",
    "time" : {
      "start" : 1576601905890,
      "stop" : 1576601905890,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "story",
    "value" : "测试3"
  }, {
    "name" : "feature",
    "value" : "测试接口"
  }, {
    "name" : "suite",
    "value" : "test_testcase"
  }, {
    "name" : "subSuite",
    "value" : "TestSingle"
  }, {
    "name" : "host",
    "value" : "anxiaodongdeMacBook-Pro.local"
  }, {
    "name" : "thread",
    "value" : "6881-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_testcase"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "fdc1287c69e94cff.json",
  "parameterValues" : [ ]
}